modules/deployment/src/main/java/org/eclipse/xpanse/modules/deployment/ServiceStateManager.java

```diff
@@ -66,32 +60,37 @@ public class ServiceStateManager {
      * @param id service id.
      * @return task id.
      */
-    public UUID startService(UUID id) {
-        ServiceStateManagementTaskType taskType = ServiceStateManagementTaskType.START;
-        DeployServiceEntity service = getDeployedServiceAndValidateState(id, taskType);
+    public ServiceOrder startService(UUID id) {
+        ServiceOrderType taskType = ServiceOrderType.SERVICE_START;
+        UUID orderId = UUID.randomUUID();
+        DeployServiceEntity service = getDeployedServiceAndValidateState(id, orderId, taskType);
         OrchestratorPlugin plugin = pluginManager.getOrchestratorPlugin(service.getCsp());
-        ServiceStateManageRequest startRequest = getServiceManagerRequest(service);
-        ServiceStateManagementTaskEntity newTask = createNewManagementTask(id, taskType);
+        ServiceStateManageRequest startRequest = getServiceManagerRequest(service, orderId);
+        ServiceOrderEntity newTask = createNewManagementTask(id, orderId, taskType);
         taskExecutor.execute(() -> asyncStartService(newTask, plugin, startRequest, service));
-        return newTask.getTaskId();
+        ServiceOrder serviceOrder = new ServiceOrder();
+        serviceOrder.setServiceId(newTask.getServiceId());
+        serviceOrder.setOrderId(newTask.getOrderId());
+        return serviceOrder;
     }
 
-    private ServiceStateManagementTaskEntity createNewManagementTask(
-            UUID serviceId, ServiceStateManagementTaskType taskType) {
-        ServiceStateManagementTaskEntity newTask = new ServiceStateManagementTaskEntity();
-        newTask.setTaskId(UUID.randomUUID());
+    private ServiceOrderEntity createNewManagementTask(
```

Comment:
use the methods available in createServiceOrderTask to create service_order entities

Reply:
fixed.